doc/man3/EVP_DigestInit.pod

```diff
@@ -386,10 +386,9 @@ B<EVP_MD>.
 
 =item EVP_MD_get_size()
 
-Return the size of the message digest when passed an B<EVP_MD> or an
-B<EVP_MD_CTX> structure, i.e. the size of the hash. A negative value or 0 can
-occur for invalid size.
-For an XOF this returns 0.
+Return the size of the message digest when passed an B<EVP_MD>, i.e. the size of
+the hash. A negative value or 0 can occur for the invalid size.
```

Comment:
Sorry, but the previous version of this sentence "A negative value or 0 can occur for invalid size." was better than "A negative value or 0 can occur for the invalid size." (the difference being `the` before `invalid` - if you must put an article there, `an` would be acceptable)